build/yamls/chart-values/antrea-aks.yml

```diff
@@ -0,0 +1 @@
+trafficEncapMode: "networkPolicyOnly"
```

Comment:
Could you add a comment in file `build/yamls/chart-values/antrea.yml`, it leads a little confusion when I see it's empty. I am unable to leave a comment to an empty file, so leave it here.